### DISCUSSION TITLE
Elección de proyecto

### DISCUSSION THREAD

**Discussion by mmasias** [mmasias/IdSw2-22-23#1] [Q&A]
Indicar aquí el [proyecto](https://github.com/mmasias/IdSw2-22-23/blob/main/docs/proyectos.md) que abordará cada [grupo](https://docs.google.com/spreadsheets/d/1MEU5q718irAnvC7s1Q-IU2q4_XMjxRl__JX2Z43gh8E/edit#gid=1508736146).

**Comment by DaniFdezC** [comment]
El grupo1 cogeremos el proyecto de la Batalla.

Daniel Fernández
José Luis Rivas
Manuel Ortega
Emilio Requena
Javier Ruiz

**Comment by AlvaroSanchez79** [comment]
El grupo2 cogeremos la maquina expendedora:
Juan Ixcopal
Andree Serrano
Omar Barrios
Alvaro Sanchez

**Reply by mmasias** [reply]
Me haría falta el nombre de usuario en git de Álvaro...

**Reply by mmasias** [reply]
No he dicho nada... :weary:

**Comment by ghost** [comment]
Grupo 3 ---> Laberinto
Ivan Santovenia
Manuel Rondon
Jaime Polidura
Alejandro Gonzalez

**Comment by hector-ae21** [comment]
Grupo 4 --> Ascensores

Hector Arrechea
Franklin López
Mauricio Soler

**Reply by hector-ae21** [reply]
David Haya no responde correo
